app.json

```diff
@@ -1,15 +0,0 @@
-{
-  "name": "WagtailBakeryDemo",
-  "description": "WagtailBakeryDemo",
-  "repository": "https://github.com/wagtail/bakerydemo",
-  "keywords": ["wagtail", "django"],
-  "env": {
-    "DJANGO_DEBUG": "off",
-    "DJANGO_SETTINGS_MODULE": "bakerydemo.settings.production",
-    "DJANGO_SECURE_SSL_REDIRECT": "on"
-  },
-  "scripts": {
-    "postdeploy": "django-admin migrate && django-admin load_initial_data && echo 'from wagtail.images.models import Rendition; Rendition.objects.all().delete()' | django-admin shell"
-  },
-  "addons": ["heroku-postgresql:hobby-dev"]
-}
```

Comment:
The reason to deploy to Heroku would be more for actually deploying it somewhere rather than just a "let me try this real quick". Especially given Heroku has removed its free tier, I think suggesting Heroku as a way to try Wagtail out isn't one that's going to get used too often. Gitpod works much better in those cases. There's little reason to deploy bakerydemo to Heroku.

Reply:
Not sure our messaging is clear on this then - we're shipping bakerydemo with a Dockerfile that mentions Heroku multiple times, and a `heroku.yml` file, but removing all the docs on how to use Heroku.

I think Heroku is still a valid and useful platform for those that want to quickly deploy bakerydemo for demoing to their team/for personal testing, and as everything else is tailored towards a Heroku deployment, why not keep it here?